tests/prague/eip7002_el_triggerable_withdrawals/test_modified_withdrawal_contract.py

```diff
@@ -0,0 +1,261 @@
+"""
+abstract: Tests [EIP-7002: Execution layer triggerable withdrawals](https://eips.ethereum.org/EIPS/eip-7002)
+    Test execution layer triggered exits [EIP-7002: Execution layer triggerable withdrawals](https://eips.ethereum.org/EIPS/eip-7002).
+
+"""  # noqa: E501
+
+from typing import List
+
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Block,
+    BlockchainTestFiller,
+    Bytecode,
+    Transaction,
+)
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_types import Requests
+
+from .helpers import (
+    WithdrawalRequest,
+    # WithdrawalRequestBase,
+    # WithdrawalRequestContract,
+    WithdrawalRequestTransaction,
+)
+from .spec import Spec as Spec_EIP7002
+from .spec import ref_spec_7002
+
+REFERENCE_SPEC_GIT_PATH: str = ref_spec_7002.git_path
+REFERENCE_SPEC_VERSION: str = ref_spec_7002.version
+
+
+def single_withdrawal_with_custom_fee(i: int) -> WithdrawalRequest:  # noqa: D103
+    return WithdrawalRequest(
+        validator_pubkey=i + 1,
+        amount=0,
+        fee=Spec_EIP7002.get_fee(0),
+    )
+
+
+@pytest.mark.parametrize(
+    "requests_list",
+    [
+        pytest.param(
+            [],
+            id="empty_request_list",
+        ),
+        pytest.param(
+            [single_withdrawal_with_custom_fee(0)],
+            id="1_withdrawal_request",
+        ),
+        pytest.param(
+            [
+                *[
+                    single_withdrawal_with_custom_fee(i)
+                    for i in range(
+                        0,
+                        15,
+                    )
+                ],
```

Comment:
Since we are already using a function to make this easier here, we can put the for loop inside the function:
```suggestion
                *withdrawal_list_with_custom_fee(15),
```

And then:
```python
def withdrawal_list_with_custom_fee(n: int) -> List[WithdrawalRequest]:  # noqa: D103
    return [WithdrawalRequest(
        validator_pubkey=i + 1,
        amount=0,
        fee=Spec_EIP7002.get_fee(0),
    ) for i in range(n)]
```
Or something similar.

Reply:
Good idea, improves readability